.gitignore

```diff
@@ -308,3 +308,4 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+.DS_Store
```

Comment:
MacOS creates this file in every folder. Is not needed at all, so no need to sync.